File: src/cn/vove7/plugin/rest/psi/RestElementType.java
package cn.vove7.plugin.rest.psi;

import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;
import cn.vove7.plugin.rest.filetype.RestLanguage;

/**
 *
 */
public class RestElementType extends IElementType {
    public RestElementType(@NotNull @NonNls String debugName) {
        super(debugName, RestLanguage.INSTANCE);
    }
}

File: src/main/kotlin/cn/vove7/plugin/rest/psi/RestElementType.kt
package cn.vove7.plugin.rest.psi

import com.intellij.psi.tree.IElementType
import cn.vove7.plugin.rest.filetype.RestLanguage
import org.jetbrains.annotations.NonNls

/**
 *
 */
class RestElementType(debugName: @NonNls String) : IElementType(debugName, RestLanguage)